### PR TITLE
Apply black to Tests/common_BioSQL*

### PR DESCRIPTION
**Pull request by martonlanga**
This pull request addresses issue #2552

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

---

I've got all sorts of warnings from flake, including: `F401, F841, D102`. Not sure how to address those.

### CODE REVIEW

File: Tests/common_BioSQL.py [hunk 1]
```diff
@@ -42,9 +42,11 @@
...
     """Load the database settings from INI file."""
     if not os.path.isfile("biosql.ini"):
-        raise MissingExternalDependencyError("BioSQL test configuration"
-                                             " file biosql.ini missing"
-                                             " (see biosql.ini.sample)")
+        raise MissingExternalDependencyError(
+            "BioSQL test configuration"
+            " file biosql.ini missing"
+            " (see biosql.ini.sample)"
+        )
```
Comment (peterjc) [inline Tests/common_BioSQL.py:49]: That string can probably be joined up into two medium lines, or maybe even one line and still fit black's 88 character limit?

**Comment by peterjc** [comment]
If you're using the brand new flake8 v3.8, there are some new issues - see #2878 

Otherwise it seems flake8 is fragile at interpreting the per-file ignore settings - it seems to care about which directory you are in. They have an open bug for this.

**Comment by peterjc** [comment]
TravisCI has just finished the style checks - just the issues from #2878, so nothing to worry about :+1:

**Comment by peterjc** [comment]
Applied, thank you - and feel free to include yourself in the NEWS and CONTRIB files in a future pull request. Or ask, and we can add you now.

**Comment by martonlanga** [comment]
Will do next time, thanks!

**Pull request merged**